Security/src/AuthConsole/ApiClients/CertificateAuthorizationApiClient.cs

```diff
@@ -0,0 +1,17 @@
+using Steeltoe.Samples.AuthConsole.Models;
+
+namespace Steeltoe.Samples.AuthConsole.ApiClients;
+
+public sealed class CertificateAuthorizationApiClient(HttpClient httpClient)
```

Comment:
This and some of the other code is copied over from AuthWeb... I didn't think it was worth the overhead of adding a shared project at this time due to the changes we'd need to make for cf deployments to work

Reply:
We can't have a shared project, right? Then `cf push` no longer works.

Reply:
We _can_, but [some extra work](https://docs.cloudfoundry.org/buildpacks/dotnet-core/index.html#multiple-projects) is required